src/app/gdg-page/components/Card-Bevy.tsx

```diff
@@ -0,0 +1,52 @@
+import Button, { BUTTON_VARIANTS } from "@/components/input/Button";
+import Image from "next/image";
+
+interface CardProps {
+  header: string;
+  date: string;
```

Comment:
I don't recall exactly, but is this supposed to be string or `Date`?

Reply:
Ah we used the date in HG and then converted it to a string